### PR TITLE
NO-ISSUE: Add mainteners from assisted for packet-setup step

### PR DESCRIPTION
**Pull request by adriengentil**
Since metal team is focusing to transistion on OFCIR, we (assisted)
would like to become mainteners of the packet-setup step in order to
take care of the ansible packet flow.

### CODE REVIEW

File: ci-operator/step-registry/baremetalds/packet/setup/OWNERS [hunk 1]
```diff
@@ -1 +0,0 @@
-../OWNERS
```
Comment (derekhiggins) [inline ci-operator/step-registry/baremetalds/packet/setup/OWNERS:1]: I wonder if you should add the maintainers one level up? 
i.e. in ci-operator/step-registry/baremetalds/packet

Comment (adriengentil) [reply ci-operator/step-registry/baremetalds/packet/setup/OWNERS:1]: it makes sense, I'll update

**Review by osherdp — APPROVED** [review]

**Pull request merged**